Framework/CurveFitting/src/ParameterEstimator.cpp

```diff
@@ -7,6 +7,8 @@
 
 #include "MantidKernel/Logger.h"
 
+#include <Poco/Timer.h>
```

Comment:
Could we change this to `#include <Poco/Mutex.h>`, we don't need the Timer I don't think?